src/App.jsx

```diff
@@ -1,33 +1,60 @@
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+const preparedFilter = (movie, query) => movie
+  .trim()
+  .toLowerCase()
+  .includes(
+    query
+      .trim()
+      .toLowerCase(),
+  );
```

Comment:
```suggestion
const checkQuery = (string, query) => {
  const preparedQuery =  query.trim().toLowerCase();
  
  return string
    .trim()
    .toLowerCase()
    .includes(preparedQuery );
};
```
As I see you pass the `prepearedQuery` to the current function, so you do it twice.
It's better to it just in this function so we don't need to prepare it and use it in every case.
Also, the function name and that it is in one line not so good for understanding.